merlin/models/tf/inputs/embedding.py

```diff
@@ -244,6 +245,39 @@ def __init__(
         self.sequence_combiner = sequence_combiner
         self.supports_masking = True
 
+    def select_by_tag(self, tags: Union[Tags, Sequence[Tags]]) -> Optional["EmbeddingTable"]:
+        """Select features in EmbeddingTable by tags.
+
+        Since an EmbeddingTable can be a shared-embedding table, this method filters
+        the schema for features that match the tags.
+
+        If none of the features match the tags, it will return None.
+
+        Parameters
+        ----------
+        tags: Union[Tags, Sequence[Tags]]
+            A list of tags.
+
+        Returns
+        -------
+        An EmbeddingTable if the tags match. If no features match, it returns None.
+        """
+        if not isinstance(tags, collections.Sequence):
+            tags = [tags]
+
+        config = self.get_config()
+        schema = config.pop("schema", None)
```

Comment:
could these 6 lines could be replaced by `selected_schema = self.schema.select_by_tag(tags)`, since the schema is always going to be set on the embedding table?

Reply:
Yes, thank you!